pkg/network/ovn_kubernetes.go

```diff
@@ -1624,18 +1624,24 @@ func bootstrapOVN(conf *operv1.Network, kubeClient cnoclient.Client, infraStatus
 		prepullerStatus.Progressing = daemonSetProgressing(prePullerDaemonSet, true)
 	}
 
-	ipsecHostDaemonSet := &appsv1.DaemonSet{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "DaemonSet",
-			APIVersion: appsv1.SchemeGroupVersion.String(),
-		},
-	}
-	nsn = types.NamespacedName{Namespace: util.OVN_NAMESPACE, Name: "ovn-ipsec-host"}
-	if err := kubeClient.ClientFor("").CRClient().Get(context.TODO(), nsn, ipsecHostDaemonSet); err != nil {
+	ipsecHostDaemonSet, err := kubeClient.Default().Kubernetes().AppsV1().DaemonSets(util.OVN_NAMESPACE).Get(context.TODO(), util.OVN_IPSEC_HOST, metav1.GetOptions{})
```

Comment:
any specific reason why ovn-ipsec-host DS retrieved twice from API ? we could just try retrieve it, if it exists, set ipsecStatus parameters, otherwise set ipsecStatus to nil.

Reply:
ok, i understand it now, this sets up `ipsecStatus` appropriately.